themes/ddbasic/sass/components/menu.scss

```diff
@@ -559,3 +559,43 @@ ul.topbar-menu {
     }
   }
 }
+
+// ==========================================================================
+// User mobile menu placeholder - lazy loading
+// ==========================================================================
+.mobile-user-menu .lazy-pane-placeholder {
```

Comment:
This seems to set a new spinner specifically for this element. Is that really necessary?

Reply:
It's due to the way these spinners is constructed. I did not see any other way to get the spinner here without having to refactor the SASS. Which would be outside the scope of this issue.